tests/garage/tf/policies/test_maml_policy.py

```diff
@@ -0,0 +1,40 @@
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.policies.gaussian_mlp_policy_with_model import GaussianMLPPolicyWithModel
+from garage.tf.policies import MamlPolicy
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyBoxEnv
+
+
+class TestMamlPolicies(TfGraphTestCase):
+    def test_maml_policy(self):
+        box_env = TfEnv(DummyBoxEnv())
+        with self.sess.as_default():
+            policy = GaussianMLPPolicyWithModel(
```

Comment:
using a real model class makes this an integration test.

the tests will be simpler to write if you mock out `policy` (or use something more like a dummy policy) and make some assertions about the relationship between MamlPolicy and its wrapped_policy

Reply:
I used a model because I would like to test the gradient flow.

Reply:
i recommend you test almost everything with mocking, and then add a smaller test with the full gradient. it will probably save you a lot of time over testing everything with full gradients

see for instance https://github.com/rlworkgroup/garage/blob/newer-logger/tests/garage/logger/test_tensor_board_output.py